articles/140_topic_and_service_name_mapping.md

```diff
@@ -265,7 +335,7 @@ Here are some examples of translations between ROS topic names and the correspon
 | `/_foo/_bar`      | Topic     | `rt___foo___bar`       |
 | `/_foo/_bar/_baz` | Topic     | `rt___foo___bar___baz` |
 
-### ROS Topic and Service Name Length Limit
+#### ROS Topic and Service Name Length Limit
```

Comment:
Talking with you offline, there is still a limit of 255, which is the partition name + the topic name in DDS. So this section gets simpler, but it still deserves to be put into the proposal (i.e. not just in the alternatives section). It is also affected by the "ros prefix", because that also eats into the topic length limit.